syntax = "proto3";

package server;

import "Protobufs.proto"; 

option java_package = "com.inventory.service";
option java_outer_classname = "ServiceProtos";

message CalculateShipmentPriceRequest {
    repeated Protobufs.Shipment shipments = 1;
}

message CalculateShipmentPriceResponse {
    map<int32, double> total_prices = 1;
}

service ShipmentPriceCalculator {
    rpc CalculateShipmentPrice (CalculateShipmentPriceRequest) returns (CalculateShipmentPriceResponse);
}
